### DISCUSSION TITLE
Will the user initiating the approval request also need powerautomate and powerapp licenses?

### DISCUSSION THREAD

**Discussion by barcafan** [microsoft/powercat-business-approvals-kit#192] [Q&A]
What are the licensing requirements for users who are just requesting the approval? 
Does the user, who initiates the approvals process also need powerapps and powerautomate licenses?

**Comment by girishuppal** [comment]
Depends on how the user triggers the approval request. If user triggers from model driven / canvas app, then the user will definitely have standard license for accessing the app.

Please read this table for more details:

![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/17232756/c0653f41-5e6b-419d-912c-4ad3ac4b5537)

**Reply by girishuppal** [reply]
Read this in conjunction as well 

https://learn.microsoft.com/en-us/power-automate/guidance/business-approvals-templates/setup#persona-licenses

![image](https://github.com/microsoft/powercat-business-approvals-kit/assets/17232756/c1319b4d-00de-4cd1-a01b-ef2cb2275874)
